qiskit/circuit/library/standard_gates/u1.py

```diff
@@ -177,6 +177,22 @@ class CU1Gate(ControlledGate):
     This is a diagonal and symmetric gate that induces a
     phase on the state of the target qubit, depending on the control state.
 
+    .. warning::
+
+       This gate is deprecated. Instead, the following replacement should be used
+
+       .. math::
+
+           CU1(\lambda, 2) = CP(\lambda, 2)
+
+       .. code-block:: python
+
+          circuit = QuantumCircuit(3)
+          circuit.cp(lambda, [0, 1], 2)
```

Comment:
It might be less confusing to just apply a single gate instead of broadcasting it, if we want to offer a simple example 🙂 
```suggestion
          circuit = QuantumCircuit(2)
          circuit.cp(lambda, 0, 1)
```